remote_command_execution_vulnerability.py

```diff
@@ -24,19 +29,31 @@
 import hashlib
 import platform
 import socket
+import urllib.parse
+import socket
 
-# make sure that script.sh on windows uses \n
+#------------------------------------------------------------------------------------------------------------------
 if platform.system() == "Windows":
-    with open("script.sh", "rt", encoding = "UTF-8") as f:
-        content = f.read()
-    with open("script.sh", "wt", encoding = "UTF-8", newline="\n") as f:
-        f.write(content)
-
-router_ip_address="miwifi.com"
-#router_ip_address = "192.168.31.1"
-router_ip_address = input("Router IP address [press enter for using the default '{}']: ".format(router_ip_address)) or router_ip_address
-
-# get stok
+    sys.exit("Stopping: script can only be run on a Mac/Linux system")

```

Comment:
Is this intentional? Windows support is being dropped again?

Reply:
If you have a Windows system, remove the code and try it. I have no Windows that was working with it, and when I have to give it to somebody without much knowledge, a live disk in VMWare would be easier.